src/Service.Tests/SqlTests/RestApiTests/RestBodyFlexibleModeTests/MsSqlRestBodyNonStrictModeTests.cs

```diff
@@ -0,0 +1,81 @@
+        {
+            {
+                "InsertOneWithNonExistingFieldInRequestBody",
+                $"SELECT [categoryid], [pieceid], [categoryName],[piecesAvailable]," +
+                $"[piecesRequired] FROM { _Composite_NonAutoGenPK_TableName } " +
+                $"WHERE [categoryid] = 3 AND [pieceid] = 1 AND [categoryName] = 'SciFi' " +
+                $"AND [piecesAvailable] is NULL AND [piecesRequired] = 1 " +
+                $"FOR JSON PATH, INCLUDE_NULL_VALUES, WITHOUT_ARRAY_WRAPPER"
+            },
+            {
+                "PutOneWithPKFieldsInRequestBody",
+                $"SELECT [categoryid], [pieceid], [categoryName], [piecesAvailable]," +
+                $"[piecesRequired] FROM { _Composite_NonAutoGenPK_TableName } " +
+                $"WHERE [categoryid] = 2 AND [pieceid] = 1 AND [categoryName] = 'SciFi' " +
+                $"AND [piecesAvailable] = 10  AND [piecesRequired] = 5 " +
+                $"FOR JSON PATH, INCLUDE_NULL_VALUES, WITHOUT_ARRAY_WRAPPER"
+            },
+            {
+                "PutOneWithNonExistingFieldInRequestBody",
+                $"SELECT [categoryid], [pieceid], [categoryName], [piecesAvailable]," +
+                $"[piecesRequired] FROM { _Composite_NonAutoGenPK_TableName } " +
+                $"WHERE [categoryid] = 2 AND [pieceid] = 1 AND [categoryName] = 'SciFi' " +
+                $"AND [piecesAvailable] = 10  AND [piecesRequired] = 5 " +
+                $"FOR JSON PATH, INCLUDE_NULL_VALUES, WITHOUT_ARRAY_WRAPPER"
+            },
+            {
+                "PatchOneWithPKFieldsInRequestBody",
+                $"SELECT [categoryid], [pieceid], [categoryName],[piecesAvailable]," +
+                $"[piecesRequired] FROM { _Composite_NonAutoGenPK_TableName } " +
+                $"WHERE [categoryid] = 1 AND [pieceid] = 1 AND [categoryName] = 'SciFi' " +
+                $"AND [piecesAvailable] is NULL AND [piecesRequired] = 0 " +
+                $"FOR JSON PATH, INCLUDE_NULL_VALUES, WITHOUT_ARRAY_WRAPPER"
+            },
+            {
+                "PatchOneWithNonExistingFieldInRequestBody",
+                $"SELECT [categoryid], [pieceid], [categoryName],[piecesAvailable]," +
+                $"[piecesRequired] FROM { _Composite_NonAutoGenPK_TableName } " +
+                $"WHERE [categoryid] = 1 AND [pieceid] = 1 AND [categoryName] = 'SciFi' " +
+                $"AND [piecesAvailable] is NULL AND [piecesRequired] = 0 " +
+                $"FOR JSON PATH, INCLUDE_NULL_VALUES, WITHOUT_ARRAY_WRAPPER"
+            }
+        };
+
+        [ClassInitialize]
+        public static async Task SetupDatabaseAsync(TestContext TestContext)
+        {
+            DatabaseEngine = TestCategory.MSSQL;
+
+            // Set rest.request-body-strict = false to simulate scenario when we operate in non-strict mode for fields in request body.
+            await InitializeTestFixture(context: null, isRestBodyStrict: false);
```

Comment:
do you  not need to pass `TestContext` here?

Reply:
Yes, this is required: https://learn.microsoft.com/en-us/visualstudio/test/using-microsoft-visualstudio-testtools-unittesting-members-in-unit-tests?view=vs-2022#class

> The methods marked with these attributes should be defined as static void or static Task, in a TestClass, and appear only once. The initialize part requires one argument of type [TestContext](https://learn.microsoft.com/en-us/dotnet/api/microsoft.visualstudio.testtools.unittesting.testcontext) and the cleanup no argument.

Reply:
To clarify, because TestContext is required, why is `null` being used when calling ` InitializeTestFixture(context: null, isRestBodyStrict: false);`

Reply:
Not required to have this param at all in the `InitializeTestFixture` method. Removed it.